Common/Prey/GameDll/ark/dialog/ArkGlobalFacts.h

```diff
@@ -38,6 +38,7 @@ class ArkGlobalFacts : public IGameTokenEventListener, public IArkObjectiveState
 	virtual void OnPsiChanged();
 	virtual void OnStatusChanged(uint64_t _statusId, bool _bActive);
 	virtual void OnAbilityAdded(const uint64_t _abilityID);
+    virtual void OnBecomeAlien();
```

Comment:
I don't see `OnBecomeAlien` in the PDB. It should be inherited from `IArkAbilityListener` so it shouldn't be needed here.

![image](https://github.com/user-attachments/assets/a71736e2-65b5-4b2f-8c16-2600e81aa502)